examples/simple/posix/UB-PTHREAD4-bad.output

```diff
@@ -0,0 +1,12 @@
+Error: UB-PTHREAD4
+Description: Attempting to lock a PTHREAD_MUTEX_DEFAULT mutex that is already locked by the current thread.
+Locked here:
+  at pthread_mutex_lock(UB-PTHREAD4-bad.c:15)
+  at locker_thread(UB-PTHREAD4-bad.c:15)
+  at <file-scope>(<unknown>)
+
```

Comment:
same here: is the whiteline necessary, or follows some more general convention?  if not, i would remove it.